File: TextBlockHighlighterCS/Extensions.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TextBlockHighlighterCS
{
    public static class Extensions
    {
        public static Boolean Contains(this string str, string txt, StringComparison c)
        {
            if (txt != null) return str.IndexOf(txt, c) >= 0;
            return false;
        }
    }
}

File: TextBlockHighlighterVB/Extensions.vb
Imports System.Runtime.CompilerServices

Module Extensions
    <Extension()>
    Function Contains(ByVal str As String, ByVal txt As String, ByVal c As StringComparison) As Boolean
        If txt IsNot Nothing Then Return str.IndexOf(txt, c) >= 0
        Return False
    End Function
End Module
